platform/feature-api/api/rdf-spec/src/data-factory.ts

```diff
@@ -69,25 +69,38 @@ export class DataFactorySpec<OutQuad extends BaseQuad = Quad> {
                 it("should return true if termType and value are equal", () => {
                     const id = "b1";
                     const term = this.dataFactory.blankNode(id);
-                    const mock: any = { termType: "BlankNode", value: id };
 
-                    assert.equal(term.equals(mock), true);
+                    const expectedMock: BlankNode = {
+                        termType: "BlankNode",
+                        value: id,
+                        equals: () => true,
```

Comment:
What does this do?

Reply:
in order to have the type here we need to provide a mocked `equals` to represent the real object

Reply:
Ah, I see!